geniza/templates/snippets/theme_toggle.html

```diff
@@ -0,0 +1,12 @@
+{% load i18n static webpack_loader %}
+
+{% block extrascript %}
+    <script defer src="{% static 'js/theme-toggle.js' %}" nonce="{{ request.csp_nonce }}"></script>
+{% endblock %}
+
+<label id="theme-toggle" for="theme-toggle-checkbox" tabindex="0">
+    {# Translators: label for dark mode/light mode theme switcher #}
+    <span class="sr-only">{% translate "Enable dark mode" %}</span>
```

Comment:
that probably shouldn't be the default text, since depending on user preferences it could start in dark mode; maybe something more general, like switch between dark and light mode?

Reply:
I was wondering about this, and that was actually similar to my original text for this. But thinking through the inputs, since it's a checkbox, I wondered if someone using a screen reader might understand "Checkbox labeled 'Enable dark mode,' checked" better than "Checkbox labeled 'Switch between light and dark mode,' checked". But I'm not sure if my thought process makes sense when you consider what the actual user input looks like in the UI.

Reply:
Ah, got it. Well, this seems ok for now — I think we'll get design input from Gissoo on it at some point, and can refine it then.